core-services/jira-lifecycle-plugin/config.yaml

```diff
@@ -290,6 +290,11 @@ orgs:
             release_notes_default_text: "Cause: What actions or circumstances cause this bug to present.\r\nConsequence: What happens when the bug presents.\r\nFix: What was done to fix the bug.\r\nResult: Bug doesn’t present anymore."
   openshift:
     repos:
+      etcd:
+        branches:
+          '*':
```

Comment:
Can you change this to release-4.20? And then when we branch change it to release-4.21?

As it is now it's going to forego that logic on all branches and we wouldn't want that.

Reply:
how about adding it for all future branches, and we drop off the last released version after GA?

Reply:
sure that works